packages/dds/tree/src/feature-libraries/chunked-forest/codec/codecs.ts

```diff
@@ -12,21 +12,97 @@ import {
 	type IJsonCodec,
 	makeVersionedValidatedCodec,
 } from "../../../codec/index.js";
-import { CursorLocationType, type SchemaAndPolicy } from "../../../core/index.js";
+import {
+	CursorLocationType,
+	type FieldKey,
+	type ITreeCursorSynchronous,
+	type SchemaAndPolicy,
+	type TreeChunk,
+	type TreeNodeSchemaIdentifier,
+} from "../../../core/index.js";
 import type { JsonCompatibleReadOnly } from "../../../util/index.js";
-import { TreeCompressionStrategy } from "../../treeCompressionUtils.js";
+import {
+	TreeCompressionStrategy,
+	TreeCompressionStrategyExtended,
+	type TreeCompressionStrategyInternal,
+} from "../../treeCompressionUtils.js";
 
 import { decode } from "./chunkDecoding.js";
 import type { FieldBatch } from "./fieldBatch.js";
 import { EncodedFieldBatch, validVersions } from "./format.js";
 import { schemaCompressedEncode } from "./schemaBasedEncode.js";
 import { uncompressedEncode } from "./uncompressedEncode.js";
+import { Type, type Static } from "@sinclair/typebox";
+
+/**
+ * Reference ID for a chunk that is incrementally encoded.
+ */
+export const ChunkReferenceId = Type.Number({ multipleOf: 1, minimum: 0 });
```

Comment:
If you want strong typing (which I think would be nice here), you can use `brandedNumberType`

Something like
```typescript
export type ChunkReferenceId  = Brand<number, "forest.ChunkReferenceId">;
const ChunkReferenceId  = brandedNumberType<ChunkReferenceId >({ multipleOf: 1, minimum: 0 });
```

then use `brand` where you initially convert from a number to a ChunkReferenceId, and typescript will make sure it doesn't get mixed up with other numbers.